docker-compose-github.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   slurm:
-    image: giovtorres/docker-centos7-slurm:20.11.8
+    image: giovtorres/docker-centos7-slurm:21.08.0
```

Comment:
@rezib I pushed a `21.08.6` version you can retry with.

Reply:
Thank you @giovtorres :+1: